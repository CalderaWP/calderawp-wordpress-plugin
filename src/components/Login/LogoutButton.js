import React from 'react';
import {Button} from '@wordpress/components';

export const LogoutButton = ({onLogout}) => {
	return (<Button
		onClick={
			(event) => {
				event.preventDefault();
				onLogout();
			}
		}
	>Logout
	</Button>);
};