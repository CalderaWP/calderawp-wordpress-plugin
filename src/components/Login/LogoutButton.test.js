import renderer from "react-test-renderer";
import React from "react";
import "@babel/polyfill";

import {LogoutButton} from "./LogoutButton";

describe( 'Logout button', () => {
	it( 'matches snapshot', () => {
		const onLogout = jest.fn();
		expect( renderer.create(
			<LogoutButton
				onLogout={onLogout}
			/>
		)).toMatchSnapshot();
	});
})
