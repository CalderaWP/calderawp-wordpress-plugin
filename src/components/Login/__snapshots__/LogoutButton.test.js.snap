// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout button matches snapshot 1`] = `
<button
  className="components-button"
  disabled={undefined}
  onClick={[Function]}
  type="button"
>
  Logout
</button>
`;
